docs/local_development.md

```diff
@@ -0,0 +1,61 @@
+# Local Development with MediaFlux
+
+This README describes the recommended approach to get up and running with MediaFlux for local development on your own machine, using Docker.
+
+## Prerequisites
+
+* Docker 
+
+## Getting started
+
+1. Make sure Docker is installed and running locally.
+1. Pull the latest TigerData Docker image from either the vendor or the private Princeton Docker registry.  Information for how to connect and pull this image are located in LastPass.
+1. In a terminal, run the following Docker command:
+    ```bash 
+      docker run --interactive --rm --tty --privileged --init --name mediaflux --publish 0.0.0.0:8888:8888 $DOCKER_IMAGE_REGISTRY/$NAMESPACE/developer-image:latest /bin/bash
+    ```
+    Where `$DOCKER_IMAGE_REGISTRY` and `$NAMESPACE` are replaced with values found in LastPass in the previous step.
+1. Upon running the Docker command, you should see output like the following:
+   ```bash 
+     WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
+     root@123abc:/setup#
+   ```
+   You are now inside of the Docker container running MediaFlux.
+1. Inside the container, run the following command:
+   ```bash 
+     /usr/bin/env java -jar /usr/local/mediaflux/bin/aserver.jar application.home=/usr/local/mediaflux nogui
+   ```
+   This will start the MediaFlux service.  Leave the service running in this tab.
+
```

Comment:
I find it simpler to not run the docker in interactive mode.  It will then just start up mediaflux.
```suggestion
    ```bash 
      docker run --publish 0.0.0.0:8888:8888 $DOCKER_IMAGE_REGISTRY/$NAMESPACE
    ```
    Where `$DOCKER_IMAGE_REGISTRY` and `$NAMESPACE` are replaced with values found in LastPass in the previous step.
1. Upon running the Docker command, you should see output like the following:
   ```bash 
     WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
     root@123abc:/setup#
   ```
    This will start the MediaFlux service.  Leave the service running in this tab.